test/src.js

```diff
@@ -299,6 +299,33 @@ describe('.src()', function() {
     ], done);
   });
 
+  it('streams a file changed after since (when file copy-pasted)', function(done) {
+    var testFilePath = inputPath + 'copy-paste-test';
+    var stat = fs.statSync(inputPath);
+    var fd, lastUpdateDate;
+
+    // When in OS user copy-paste file, its mtime does not changed
```

Comment:
Instead of emulating this, can you use `child_process` to copy a file?  You can create separate tests for each OS